evap/evaluation/tools.py

```diff
@@ -122,3 +123,21 @@ def clean_email(email):
             if email.endswith(original_domain):
                 return email[:-len(original_domain)] + replaced_domain
     return email
+
+
+def export_contributor_results(contributor):
+    from evap.evaluation.models import CourseType, Degree, Semester
+    from evap.results.exporters import ExcelExporter
+
+    filename = "Evaluation_{}.xls".format(contributor.full_name)
+    response = HttpResponse(content_type="application/vnd.ms-excel")
+    response["Content-Disposition"] = "attachment; filename=\"{}\"".format(filename)
+    ExcelExporter().export(
+        response,
+        Semester.objects.all(),
+        [(Degree.objects.all(), CourseType.objects.all())],
+        include_not_enough_voters=True,
```

Comment:
that's surprising, can contributors usually access those?

Reply:
when collecting results we always check for `can_publish_rating_results`. if it is false, the results will not be returned. so if an evaluation does not have enough voters, it will here be included into the export, but with an empty column.
i would expect this behavior as a contributor in contrast to not finding the evaluation at all.